src/core/tests/queryClient.test.tsx

```diff
@@ -137,13 +137,13 @@ describe('queryClient', () => {
     test('should create a new query if query was not found', () => {
       const key = queryKey()
       queryClient.setQueryData(key, 'bar')
-      expect(queryClient.getQueryData(key)).toBe('bar')
+      expect(queryCache.find(key)).not.toBe(undefined)
```

Comment:
why do we need to change that test - what was wrong with the previous assertion?

Reply:
getQueryData returns undefined for existing idle queries or non-existing queries. I wanted to ensure query was actually created.